sklearn/manifold/sammon.py

```diff
@@ -0,0 +1,311 @@
+"""
+Sammon's Mapping
+"""
+
+# author: Poom Chiarawongse <eight1911@gmail.com>
+# License: BSD
+
+# """
+# He did not stop outside the door but walked quickly into the yard. His soul
```

Comment:
?????

Reply:
I thought it would be nice to put a quote I like from a book as the comment at the top for people who come to read the implementation details. But now that you point it out, it's a little stupid.